flake.nix

```
@@ -40,32 +45,34 @@
           gettext
           texinfo
         ];
-    in {
-      devShell = pkgs.mkShell {
-        buildInputs = build_deps ++ gnu_testing_deps;
+      in {
+        default = pkgsFor.${system}.pkgs.mkShell {
+          packages = build_deps ++ gnu_testing_deps;
 
-        RUSTC_VERSION = "1.75";
-        LIBCLANG_PATH = pkgs.lib.makeLibraryPath [pkgs.llvmPackages_latest.libclang.lib];
-        shellHook = ''
-          export PATH=$PATH:''${CARGO_HOME:-~/.cargo}/bin
-          export PATH=$PATH:''${RUSTUP_HOME:-~/.rustup}/toolchains/$RUSTC_VERSION-x86_64-unknown-linux-gnu/bin/
-        '';
+          RUSTC_VERSION = "1.75";
```

Let's take advantage of this PR to bump the rust version to the current MSRV
```suggestion
          RUSTC_VERSION = "1.79";
```

---

Just bumped to 1.82